packages/backend/src/controllers/userController.ts

```diff
@@ -51,7 +53,11 @@ export class UserController extends BaseController {
      * Get authenticated user
      * @param req express request
      */
-    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @Middlewares([
+        authenticateServiceAccount([SystemServiceAccountScope.SYSTEM_LOGIN]),
```

Comment:
This is not needed anymore, check the latest version of https://github.com/lightdash/lightdash/pull/15354 . If you created a branch from there, you might want to sync.

Reply:
Ah I grabbed the other branch